### PR TITLE
fix: Keyboard covering input fields in the feedback screen

### PR DESCRIPTION
**Pull request by fnKode**
If you select any input field in the feedback menu then you should now be able to scroll to the bottom of the page while also seeing the keyboard.

### CODE REVIEW

File: packages/core/src/components/screens/create-incident/CreateIncidentScreen.tsx [hunk 1]
```diff
@@ -72,64 +72,66 @@ export const CreateIncidentScreen = () => {
...
-                    <View style={[styles.popupBox, styles.popupSuccess]}>
-                        <Typography>{`Ticket number: ${ticketNumber}`}</Typography>
-                    </View>
-                )}
-                {error && (
-                    <View style={[styles.popupBox, styles.popupDanger]}>
-                        <Typography>
-                            {/* eslint-disable-next-line @typescript-eslint/no-base-to-string,@typescript-eslint/restrict-template-expressions -- this rule sucks */}
-                            {`An error occurred creating your ticket: ${error}`}
+        <KeyboardAvoidingView behavior="padding" keyboardVerticalOffset={115}>
```
Comment (tormodAase) [inline packages/core/src/components/screens/create-incident/CreateIncidentScreen.tsx:75]: The `keyboardVerticalOffset` value seems a bit arbitrary. Does this solution work on all devices? iPhone 8, iPad mini, iPad Pro etc

**Review by tormodAase — DISMISSED** [review]
Looks good! Take a look at my comment

File: packages/core/src/components/screens/create-incident/CreateIncidentScreen.tsx (hunk 1 above)
Comment (fnKode) [reply packages/core/src/components/screens/create-incident/CreateIncidentScreen.tsx:75]: This is what we are using in notifications at least, and it seems to work fine for all the devices I have tried. But I will test a bit and verify that it works here as well 👍

Comment (fnKode) [reply packages/core/src/components/screens/create-incident/CreateIncidentScreen.tsx]: I tested some iphones and ipads and the styling was a bit weird, but I found a fix by adding `flex:1` to the `KeyboardAvoidingView`. Now it looks good on the two ipads and the two iphones 👍

**Review by tormodAase — APPROVED** [review]

**Pull request merged**